src/index.html

```diff
@@ -1,14 +1,28 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width, 
+        user-scalable=no, 
+        initial-scale=1.0, 
+        maximum-scale=1.0, 
+        minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="list">
+      <div class="list__item list__item1">1</div>
+      <div class="list__item list__item2">2</div>
+      <div class="list__item list__item3">3</div>
+      <div class="list__item list__item4">4</div>
+      <div class="list__item list__item5">5</div>
+      <div class="list__item list__item6">6</div>
+    </div>
```

Comment:
```suggestion
      <div class="list__item">1</div>
      <div class="list__item">2</div>
      <div class="list__item">3</div>
      <div class="list__item">4</div>
      <div class="list__item">5</div>
      <div class="list__item">6</div>
    </div>
```
Используй `nth-child`